.github/workflows/workflow.yml

```diff
@@ -301,14 +345,66 @@ jobs:
           init_test_run $HOME/.linkerd
           ${{ matrix.integration_test }}_integration_tests
         )
+  
+  kind_integration_gh:
+    strategy:
+      matrix:
+        integration_test: [deep, upgrade, helm, custom_domain, external_issuer]
```

Comment:
it's a bummer to have separate `kind_integration_host` and `kind_integration_gh`. i think one particular challenge is dynamically setting the `max-parallel` value. i _think_ this will work:

```diff
+  set_max_parallel:
+    needs: [docker_pull, docker_build, kind_setup]
+    steps:
+    - set_max_parallel_gh:
+      if: github.event_name != 'pull_request' || !github.event.pull_request.head.repo.fork
+      run: echo "::set-output name=max_parallel::5"
+    - set_max_parallel_host:
+      if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork
+      run: echo "::set-output name=max_parallel::3"
+
   kind_integration_host:
     strategy:
-      max-parallel: 3
+      max-parallel: {{ jobs.set_max_parallel.steps.outputs.max_parallel}}
       matrix:
         integration_test: [deep, upgrade, helm, custom_domain, external_issuer]
-    needs: [docker_pull, docker_build, kind_setup]
+    needs: [set_max_parallel]
     name: Integration tests (${{ matrix.integration_test }})
     runs-on: ubuntu-18.04
     steps:
```

...going a step further, we should ideally be able to consolidate `kind_integration_host` and `kind_integration_gh`, and just switch on setting `DOCKER_HOST` and doing the custom kubectl/ssh/tunnel/foo for non-forked builds. This could probably work with either `engineerd/setup-kind@v0.3.0`, or the existing setup. Which approach matters less, more important they use the same jobs/steps/tasks.

Reply:
This is an awesome idea, but unfortunately it's not going to work out. Right now the only way to pass variables between jobs are the `{upload, download}-artifact` actions.

It would be cool to write the `max-parallel` value to a file and read it in a later job, but that also will not work out as a *step* would need to be reading the downloaded artifact.